### PR TITLE
fix field clashes with 'file' in Django 1.9

### PR DESCRIPTION
**Pull request by stefanfoulis**
Set cmsplugin_ptr and remove related_name to avoid field name clashes
with any other plugin that has a field called "file".

see https://github.com/divio/django-cms/issues/5030

### CODE REVIEW

File: djangocms_text_ckeditor/models.py [hunk 1]
```diff
@@ -23,6 +23,13 @@ def hyphenate(t):
 @python_2_unicode_compatible
 class AbstractText(CMSPlugin):
     """Abstract Text Plugin Class"""
+
+    # Set cmsplugin_ptr and remove related_name to avoid field name clashes
+    # with any other plugin that has a field called "file".
```
Comment (czpython) [inline djangocms_text_ckeditor/models.py]: a field called "abstracttext"?

Comment (stefanfoulis) [inline djangocms_text_ckeditor/models.py]: no, a field called whatever the lowercase name of the concrete subclass is called. In this case ``text``.

Comment (stefanfoulis) [inline djangocms_text_ckeditor/models.py]: and even if it were ``abstracttext``... still a field name you might want to use ;-)

**Comment by czpython** [comment]
:+1:

**Pull request merged**